integration/v3_grpc_inflight_test.go

```diff
@@ -75,3 +76,38 @@ func TestV3MaintenanceDefragmentInflightRange(t *testing.T) {
 
 	<-donec
 }
+
```

Comment:
there were some changes to the mvcc code so `TestV3MaintenanceHashInflight` would appear to work. Namely,  `TestStoreHashAfterForceCommit` and the `stopc` logic in `Hash` should probably be removed.